enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/PresenceCheckService.java

```diff
@@ -134,13 +134,14 @@ public void checkPresenceVerification(
         final PresenceCheckResult result = presenceCheckProvider.getResult(ownerId, sessionInfo);
         auditService.auditPresenceCheckProvider(idVerification, "Got presence check result: {} for user: {}", result.getStatus(), ownerId.getUserId());
 
+        // Evaluate the result
         if (result.getStatus() != PresenceCheckStatus.ACCEPTED) {
             logger.info("Not accepted presence check, status: {}, {}", result.getStatus(), ownerId);
-            evaluatePresenceCheckResult(ownerId, idVerification, result);
-            return;
+        } else {
```

Comment:
Does the code need to drop the `evaluatePresenceCheckResult(ownerId, idVerification, result);` line? I understand we do not want to return and want to continue uploading the photo, but we can still proceed with the business logic (which we would miss in the case the code throws on `if (photo == null)` line)... or alternatively, we should add the `evaluatePresenceCheckResult` call there...

Reply:
Absolutely true, thanks for the comment